n3fit/setup.py

```diff
@@ -13,6 +13,7 @@
 
         entry_points = {'console_scripts':
             ['n3fit = n3fit.scripts.n3fit_exec:main',
+             'evolven3fit = evolven3fit.evolven3fit_cli:cli',
```

Comment:
Seeing as we have a scipts folder for all the other scripts, we could follow the same convention here as well.

Reply:
So [option 2.](https://github.com/NNPDF/nnpdf/pull/1537#issuecomment-1205111113), which is also fine by me